static/app/components/multiPlatformPicker.tsx

```diff
@@ -124,10 +117,34 @@ function PlatformPicker(props: PlatformPickerProps) {
       );
     };
 
+    const popularTopOfAllCompare = (a: PlatformIntegration, b: PlatformIntegration) => {
+      // for the all category, put popular ones at the top in the order they appear in the popular list
+      if (isPopular(a) && isPopular(b)) {
+        // if both popular, maintain ordering from popular list
+        return popularIndex(a) - popularIndex(b);
+      }
+      // if one popular, that one should be first
+      if (isPopular(a) !== isPopular(b)) {
+        return isPopular(a) ? -1 : 1;
+      }
+      // since the all list is coming from a different source (platforms.json)
+      // we can't go off the index of the item in platformCategories.tsx since there is no all list
+      return a.id.localeCompare(b.id);
+    };
+
+    const serverlessCompare = (a: PlatformIntegration, b: PlatformIntegration) => {
+      // for the serverless category, sort by service, then language
+      // the format of the ids is language-service
+      const compareServices = a.id.split('-')[1].localeCompare(b.id.split('-')[1]);
```

Comment:
@vpandiarajan20 can we add some logic to handle a case where there isn't a `-` in the name?

Reply:
Don't want to assume that future platforms will always have a `-`